.github/ISSUE_TEMPLATE/gmentee_selection_template.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I am not sure we need a dedicated list here

Reply:
Okay, I can add it to the `gmnetor_template`.